### PR TITLE
Return folder as part of file metadata

### PR DESCRIPTION
**Pull request by MBcode**
## Review Time Estimate

- [ ] Immediately
- [ x] Within one week
- [ ] When possible

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) 
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the CHANGELOG.md.
- [ ] I have signed the CLA
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

The file /metadata route calls jsonfile, that now has a "paths": with the folder's path, or "/"
From issue [228](https://github.com/clowder-framework/clowder/issues/228)

### CODE REVIEW

File: app/api/Files.scala [hunk 1]
```diff
@@ -716,6 +716,13 @@ class Files @Inject()(
   }
 
   def jsonFile(file: File, serverAdmin: Boolean = false): JsValue = {
+    val foldersContainingFile = folders.findByFileId(file.id)
+    val allPaths: List[List[String]] =  (for (folder <- foldersContainingFile) yield (folderPath(folder, List()).tail))
+    var path_str = allPaths.map(xl => "/" + xl.map(x => x.toString()).mkString("/")).mkString("")
+    if(path_str == "")
```
Comment (robkooper) [inline app/api/Files.scala:722]: can you add the `{` on the same line as the if

File: app/api/Files.scala [hunk 2]
```diff
@@ -725,7 +732,9 @@ class Files @Inject()(
       "size" -> file.length.toString,
       "thumbnail" -> file.thumbnail_id.orNull,
       "authorId" -> file.author.id.stringify,
-      "status" -> file.status)
+      "status" -> file.status,
```
Comment (robkooper) [inline app/api/Files.scala:735]: nice touch

**Review by robkooper — CHANGES_REQUESTED** [review]
please update the CHANGELOG

File: app/api/Files.scala (hunk 2 above)
Comment (robkooper) [inline app/api/Files.scala:735]: A separate request from a user, can you add the following as well:
```suggestion
      "views" -> file.stats.views,
      "downloads" -> file.stats.downloads,
      "status" -> file.status,
```

File: app/api/Files.scala (hunk 1 above)
Comment (MBcode) [reply app/api/Files.scala:722]: done as: if(path_str == "") {

File: app/api/Files.scala (hunk 2 above)
Comment (MBcode) [reply app/api/Files.scala:735]: tnx

**Comment by MBcode** [comment]
For issue [228](https://github.com/clowder-framework/clowder/issues/228)
api.Files jsonfile, adds to fields "downloads" and "views"
added to CHANGELOG.md
Will create a new issue, to turn above toString() fields into int, along with "size" which in now also a string

**Comment by robkooper** [comment]
did you intend to close this issue?

**Review by robkooper — APPROVED** [review]

**Pull request merged**